### PR TITLE
Update nodejs examples and docs

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Update nodejs examples and docs

## Links to any relevant issues

https://github.com/iotaledger/wallet.rs/issues/1257

## Type of change

- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

## How the change has been tested

Running the examples and yarn start in the documentation folder

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: bindings/nodejs/examples/25-mint-nft.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    try {
+        const manager = await getUnlockedManager();
+
+        const account = await manager.getAccount('0');
+
+        await account.sync();
+
+        const response = await account.mintNfts([
+            {
+                // Hello in bytes
```
Comment (thibault-martinez) [inline bindings/nodejs/examples/25-mint-nft.js:16]: 👋🏻 🤖

File: bindings/nodejs/lib/AccountManager.ts [hunk 2]
```diff
@@ -75,31 +85,45 @@ export class AccountManager {
...
+     * Delete all accounts and the database folder.
+     */
     async deleteAccountsAndDatabase(): Promise<void> {
         await this.messageHandler.sendMessage({
             cmd: 'DeleteAccountsAndDatabase',
         });
     }
 
+    /**
+     * Destroy the AccountManager and drop it's database connection.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/AccountManager.ts]:
```suggestion
     * Destroy the AccountManager and drop its database connection.
```

File: bindings/nodejs/lib/AccountManager.ts [hunk 3]
```diff
@@ -75,31 +85,45 @@ export class AccountManager {
...
+     */
     async generateMnemonic(): Promise<string> {
         const response = await this.messageHandler.sendMessage({
             cmd: 'GenerateMnemonic',
         });
         return JSON.parse(response).payload;
     }
 
+    /**
+     * Get an account by it's alias or index.
```
Comment (thibault-martinez) [inline bindings/nodejs/lib/AccountManager.ts]:
```suggestion
     * Get an account by its alias or index.
```

File: bindings/nodejs/types/network.ts [hunk 4]
```diff
@@ -1,16 +1,19 @@
...
 }
 
+/** Basic Auth or JWT */
 export type Auth = {
     jwt?: string;
     username?: string;
     password?: string;
 };
 
+/** Information about the tetwork and client */
```
Comment (thibault-martinez) [inline bindings/nodejs/types/network.ts]: tetwork :trollface:

File: documentation/docs/libraries/nodejs/how_to/6_send_amount.mdx [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+title: How To List Send Amount
```
Comment (thibault-martinez) [inline documentation/docs/libraries/nodejs/how_to/6_send_amount.mdx]: 🤔

File: documentation/docs/libraries/nodejs/how_to/9_mint_native_tokens.mdx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: How To Mint Native Tokens
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import MoreExamples from "./_more_examples.md";
+import mint_native_tokens from  '!!raw-loader!./../../../../../bindings/nodejs/examples/22-mint-native-tokens.js';
+
+
```
Comment (thibault-martinez) [inline documentation/docs/libraries/nodejs/how_to/9_mint_native_tokens.mdx]: 2 blank lines -> OCD trigerred

File: documentation/docs/libraries/nodejs/how_to/3_restore.mdx [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+title: How To Restore
```
Comment (thibault-martinez) [inline documentation/docs/libraries/nodejs/how_to/3_restore.mdx]: How to restore what ? Maybe not very clear

File: documentation/docs/libraries/nodejs/how_to/2_backup.mdx [hunk 8]
```diff
@@ -0,0 +1,14 @@
+---
+title: How To Backup
```
Comment (thibault-martinez) [inline documentation/docs/libraries/nodejs/how_to/2_backup.mdx]: Same

File: documentation/docs/libraries/nodejs/how_to/1_generate_addresses.mdx [hunk 9]
```diff
@@ -0,0 +1,13 @@
+---
+title: How To Generate Addresses
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import MoreExamples from "./_more_examples.md";
+import generate_address from  '!!raw-loader!./../../../../../bindings/nodejs/examples/2-generate-address.js';
```
Comment (thibault-martinez) [inline documentation/docs/libraries/nodejs/how_to/1_generate_addresses.mdx]: address or addresses ?

File: bindings/nodejs/types/account.ts [hunk 10]
```diff
@@ -50,6 +58,7 @@ export interface AccountSyncOptions {
     syncOnlyMostBasicOutputs?: boolean;
 }
 
+/** The account object */
```
Comment (thibault-martinez) [inline bindings/nodejs/types/account.ts:61]: account or account metadata ?

Comment (Thoralf-M) [reply bindings/nodejs/types/account.ts:61]: It's not account metadata, it's the account, don't know why it's called meta

Comment (thibault-martinez) [reply bindings/nodejs/types/account.ts:61]: Should we rename it ?

**Review by thibault-martinez — APPROVED** [review]

File: bindings/nodejs/types/account.ts (hunk 10 above)
Comment (Thoralf-M) [reply bindings/nodejs/types/account.ts:61]: I would say yes, but let's ask Firefly team first, they came up with this name

**Review by samuel-rufi — APPROVED** [review]

**Pull request merged**